### DISCUSSION TITLE
Is XDM dead?

### DISCUSSION THREAD

**Discussion by FrostedVolcano** [subhra74/xdm#1243] [General]
The last pre-release was on June 23, 2023, the last update on the windows store was on Jan 17, 2023. It's been more than a year since the last update. I don't know if Subhra has made any kind of public announcements about himself or the project. This is a bit worrying,
whether the developer is ok or the project is going to live.

**Comment by rollingmoai** [comment]
- https://github.com/subhra74/xdm/issues/638

**Reply by FrostedVolcano** [reply]
This is almost 3 years ago.

**Comment by Wonchong2024** [comment]
I think it's dead.

**Comment by Kedryn** [comment]
So dead...
the problem is that it's wasting new user's time, because XDM does not work anymore even for basic download.

**Reply by FrostedVolcano** [reply]
It's working totally fine for me, better than anything else.

**Reply by Kedryn** [reply]
It does not even start the scheduled dowload, for me.
Errors everywhere.

**Reply by FrostedVolcano** [reply]
Maybe try reinstalling

**Reply by nCold** [reply]
> It does not even start the scheduled dowload, for me. Errors everywhere.

am on fedora 42 workstation, and it works just fine, it grabs videos except yt sadly, and download normally, my only inconvenient is when i try to add tasks to schedule, have to click to advanced then queue then add to queue, i think it should give add to queue download now cancel
as for status of the project, the dev stated his rewriting the whole project in C#, that migration isn't going to be easy or fast, unless the dev is doing it as full time project which i suspect is not

**Reply by Kedryn** [reply]
> > It does not even start the scheduled dowload, for me. Errors everywhere.
> 
> am on fedora 42 workstation, and it works just fine, it grabs videos except yt sadly, and download normally, my only inconvenient is when i try to add tasks to schedule, have to click to advanced then queue then add to queue, i think it should give add to queue download now cancel as for status of the project, the dev stated his rewriting the whole project in C#, that migration isn't going to be easy or fast, unless the dev is doing it as full time project which i suspect is not

Be realistic.
It's been, how much, 2 years without any word from the programmer?
This project is dead and it's doomed to stop working sooner or later. Don't invest time in this until the owner gets back (and i don't think that will ever happen)

**Comment by SkywardGazing** [comment]
Sad,
XDM 8.0.29 has its share of unpolished, niggling issues, but in its half-polished state it is still the BEST all-around download manager to be had.  I prefer it to all others going back many many years.  Subhra74, PLEASE breathe some life into this project.  You've created a stand-out here.  Granted, I'm just an "end-user" so my opinion means next-to-nothing, but I've tried them all.  XDM (both the old Java & beta C#) are the best DLMs out there for Windows.  Even in its half-completed state I have been using v8.0.29 almost daily for well over a year.  I would be crushed to hear the best download manager out there is abandoned.

**Reply by e-t-l** [reply]
Has anyone made a fork of XDM?

**Reply by FrostedVolcano** [reply]
@e-t-l as far as I know, there's isn't any fork. All the other alternatives suck. It's just a bad time.

**Reply by nehmax** [reply]
This app does what it says on the tin.
Best on linux imo.

**Reply by e-t-l** [reply]
> This app does what it says on the tin.

...except for when it doesn't, according to 785 outstanding issues

**Comment by ProgMEM-CC** [comment]
Please… someone make a fork…

**Reply by Simlock** [reply]
You can try [AB Download Manager](https://github.com/amir1376/ab-download-manager)

**Comment by Kedryn** [comment]
"it is still the BEST all-around download manager"
when it works and does what it's supposed to do.

**Comment by Cloud-Mak** [comment]
![image](https://github.com/user-attachments/assets/9275550b-9ace-41d8-bb49-bde2540d5a59)

**Reply by nCold** [reply]
just ignore google's warning, by violation they mean video grabber, they class it as piracy and IP infringement, just ignore it

**Comment by H5820121** [comment]
@SkywardGazing 

> XDM (both the old Java & beta C#) are the best DLMs out there for Windows

I need to choose a download manager for myself, and I would love to hear from your experience why XDM is better than FDM or IDM?

**Comment by H5820121** [comment]
@Simlock 

> You can try [AB Download Manager](https://github.com/amir1376/ab-download-manager)

Why not FDM? I would love to hear from your experience.

**Reply by SkywardGazing** [reply]
I apologize for the delay.   For some context, I am coming up on retirement after nearly 30 years in I.T. and I am a tad set in my ways.  

At the time(s) that I went looking for download managers, my primary goal was to be be able to logon to my Microsoft TechNET/Visual Studio account and pull down large ISOs.  XDM has consistently been able to deal with downloads with associated logon credentials where I found others (like FDM) would start the logon, and choke halfway through.  Short story, it met my particular needs.  The other thing I like about XDM is that it seems to focus almost exclusively on scheduling and completing file transfers.  Other offerings there seem to want to expand their feature set to include video downloaders, and even torrent management; which is stuff which I really don't need.  If I want to rip vids, or pull down the latest Ubuntu release via bit torrent, I have other software that is better suited to those goals.   In short, XDM focused on download management.  Its also stable, reliable and fast. 

But as you can see.  This is my personal preference/opinion.  

Your goals may be completely different.  FDM is definitely an alternative you SHOULD consider; it definitely has its strong points.  But for me, I've tried a bunch of offerings out there... XDM is definitely still MY favorite.

**Comment by ngtbalajitechnology** [comment]
@subhra74 started working for Microsoft.
https://in.linkedin.com/in/subhra-das-gupta-7b3b322a
